src/include/btree_inline.h

```diff
@@ -2296,14 +2296,18 @@ __wt_btcur_skip_page(
 {
     WT_ADDR_COPY addr;
```

Comment:
I think we should add a comment to describe having a copy of addr is important. Otherwise, we may read inconsistent data from the ref if checkpoint race with us?

Reply:
The addr can be formed from either `ref->addr` or `unpack.cell`, so it must be a copy only. The addr_copy function is called in many places. I do not think adding a comment to one location will be beneficial.